docs/DeveloperGuide.md

```diff
@@ -153,6 +156,20 @@ for subsequent prompts.
 ---
 
 ### Exercise
+#### List Exercise
+
+
+User's command. If the user's command type is to list the exercises available, i.e. `exercise /list`, the
```

Comment:
Line 162, first sentence: Is the "user's command." supposed to be there xD?

Reply:
thanks for spotting that!!